lib/dartdoc.dart

```diff
@@ -22,6 +22,11 @@ import 'src/model_utils.dart';
 
 const String DEFAULT_OUTPUT_DIRECTORY = 'docs';
 
+const String NAME = 'dartdoc';
+
+// Update when puspec version changes
+const String VERSION = '0.0.1';
```

Comment:
should we have a grinder task to bump this and our pubspec.yaml ?

Also, how about a test that opens pubspec.yaml and compares to this?